ocs_ci/ocs/utils.py

```diff
@@ -887,24 +896,25 @@ def setup_ceph_toolbox(force_setup=False, storage_cluster=None):
             )
             return
 
-        # for OCS >= 4.3 there is new toolbox pod deployment done here:
-        # https://github.com/openshift/ocs-operator/pull/207/
-        log.info("starting ceph toolbox pod")
-        cmd = (
-            f"oc patch storagecluster {storage_cluster} -n {config.ENV_DATA['cluster_namespace']} --type "
-            'json --patch  \'[{ "op": "replace", "path": '
-            '"/spec/enableCephTools", "value": true }]\''
-        )
-        run_cmd(cmd)
-        toolbox_pod = OCP(
-            kind=constants.POD, namespace=config.ENV_DATA["cluster_namespace"]
-        )
-        toolbox_pod.wait_for_resource(
-            condition="Running",
-            selector="app=rook-ceph-tools",
-            resource_count=1,
-            timeout=120,
-        )
+        with config.RunWithProviderConfigContextIfAvailable():
```

Comment:
Looks like because of this change the follow up patch command is not executed on normal deployment.